app/models/knight.rb

```diff
@@ -2,14 +2,16 @@ class Knight < Piece
 
 	def valid_move?(new_x, new_y)
 		#needs refactoring, but normal is_obstructed method doesn't apply since knight "jumps" over pieces
-	  return false if squared_occupied?(new_y, new_y)
+	  return false if square_occupied?(new_y, new_y)
 	  return false if off_board?(new_x, new_y)
 	  return true if 
 	  	(x_position - new_x.to_i).abs == 1 && (y_position - new_y.to_i).abs == 2 ||
 	  	(x_position - new_x.to_i).abs == 2 && (y_position - new_y.to_i).abs == 1  
 	end 
 	  false 
 end
+
+KNIGHT = 'Knight'.freeze
```

Comment:
Not sure what the purpose of the frozen strings is...

Reply:
Left you a message about this in the slack, but I found this freeze method to be able to reference constants elsewhere in the app, such as the check logic with the King.  What's a better way that I can go about this?